public/app/features/users/state/selectors.ts

```diff
@@ -1,18 +1,18 @@
-export const getUsers = state => {
+export const getUsers = (state: any) => {
   const regex = new RegExp(state.searchQuery, 'i');
 
-  return state.users.filter(user => {
+  return state.users.filter((user: any) => {
```

Comment:
this looks strange, if users array is typed there should be no need for any here